extract_verbs.sh

```diff
@@ -3,20 +3,29 @@
 set -e
 
 cd ..
-# Download latest Italian Wikipedia dump
-wget http://download.wikimedia.org/itwiki/latest/itwiki-latest-pages-articles.xml.bz2
+# Check if the argument is null
+if [ ! -n "$1" ] ;then
+    echo "No Wikipedia dump URL entered!"
+    echo "Usage:"
+    echo "./extract_verbs.sh WikipediaDumpURL"   
+fi
+# Get the Wikimedia Dump URL
+wikimediaDumpURL=$1
+# Download latest Wikipedia dump
+wget -o wikipedia-dump.xml.bz2 $wikimediaDumpURL
```

Comment:
Hi @marfox, 
Yes, it should be a big O. My mistake. 
I have a quick question. Are these tasks in sequence or in parallel?
Thank you for your reply!